alignak/objects/config.py

```diff
@@ -1866,152 +1863,124 @@ def create_business_rules_dependencies(self):
-                mod = Module(data)
-                mod_to_add.append(mod)
+            # Ok, the user wants some monitoring logs
+            if not self.got_broker_module_type_defined('logs'):
+                msg = "Your configuration parameter '%s = %s' needs to use an external module " \
+                      "such as 'logs' but I did not found one!" % \
+                      ('log_file', self.log_file)
+                logger.error(msg)
+                self.configuration_errors.append(msg)
+            else:
+                msg = "Your configuration parameters '%s = %s' are deprecated " \
+                      "and will be ignored. Please configure your external 'logs' module " \
+                      "as expected." % \
+                      ('log_file', self.log_file)
+                logger.warning(msg)
+                self.configuration_warnings.append(msg)
 
         # Now the syslog facility
-        if self.use_syslog:
+        if hasattr(self, 'use_syslog') and self.use_syslog:
             # Ok, the user want a syslog logging, why not after all
-            got_syslog_module = self.got_broker_module_type_defined('syslog')
-
-            # We need to create the module on the fly?
-            if not got_syslog_module:
-                data = {'python_name': 'syslog',
-                        'module_alias': 'Syslog-Autogenerated'}
-                mod = Module(data)
-                mod_to_add.append(mod)
-
-        # Now the service_perfdata module
-        if self.service_perfdata_file != '':
-            # Ok, we've got a path for a service perfdata file
-            got_service_perfdata_module = self.got_broker_module_type_defined('service_perfdata')
-
-            # We need to create the module on the fly?
-            if not got_service_perfdata_module:
-                data = {'python_name': 'service_perfdata',
-                        'module_alias': 'Service-Perfdata-Autogenerated',
-                        'path': self.service_perfdata_file,
-                        'mode': self.service_perfdata_file_mode,
-                        'template': self.service_perfdata_file_template}
-                mod = Module(data)
-                mod_to_add.append(mod)
+            if not self.got_broker_module_type_defined('logs'):
+                msg = "Your configuration parameter '%s = %s' needs to use an external module " \
+                      "such as 'logs' but I did not found one!" % \
+                      ('use_syslog', self.use_syslog)
+                logger.error(msg)
+                self.configuration_errors.append(msg)
+            else:
+                msg = "Your configuration parameters '%s = %s' are deprecated " \
+                      "and will be ignored. Please configure your external 'logs' module " \
+                      "as expected." % \
+                      ('use_syslog', self.use_syslog)
+                logger.warning(msg)
+                self.configuration_warnings.append(msg)
```

Comment:
these 13 line before, you can merge with little before and test _log_file_ because same code for _log_file_ and _use_syslog_